website/docs/audit.md

```diff
@@ -3,7 +3,30 @@ id: audit
 title: Audit
 ---
 
-The audit functionality enables periodic evaluations of replicated resources against the policies enforced in the cluster to detect pre-existing misconfigurations. Audit results are stored as violations listed in the `status` field of the failed constraint.
+Audit performs periodic evaluations of existing resources against constraints, detecting pre-existing misconfigurations.
+
+## Reading Audit Results
+
+There are three ways to gather audit results, depending on the level of detail needed.
+
+### Prometheus Metrics
+
+Prometheus metrics provide an aggregated look at the number of audit violations:
+
+* `gatekeeper_audit_last_run_time` provides the timestamp of the most recently completed audit run
+* `gatekeeper_violations` provides the total number of audited violations for the last audit run, broken down by violation severity
+
```

Comment:
do we want to add `gatekeeper_audit_duration_seconds` here or link to metrics doc?